src/config.rs

```diff
@@ -49,6 +49,7 @@ pub(crate) struct Config {
     #[serde(alias = "canonicalize-issue-links")]
     pub(crate) issue_links: Option<IssueLinksConfig>,
     pub(crate) no_mentions: Option<NoMentionsConfig>,
+    pub(crate) behind_master: Option<BehindMasterConfig>,
```

Comment:
The upstream branch might not be named `master` in all repos. I would propose `behind_upstream`.